tests/test_apidir.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import shutil
+
+from docuploader import shell, tar
+from google.cloud import storage
+from google.oauth2 import service_account
+import pytest
+
+from docpipeline import generate
+
+
+@pytest.fixture
+def api_dir(tmpdir):
+    shutil.copytree("testdata", tmpdir / "api", dirs_exist_ok=True)
+    shutil.copy("testdata/docs.metadata", tmpdir)
+    return tmpdir
+
+
+def test_apidir(api_dir, tmpdir):
```

Comment:
Mind moving this into the existing test file? Looking at it now, we may be able to just use the same `api_dir` fixture -- just use `tmpdir` not `api_dir` when running `docuploader`.

If we can't use the same fixture, please update the name (`api_dir` should return the `api/` directory).

Also, there may be parts we can refactor into common helpers to avoid duplicating code.

Reply:
Fixed, merged the two test cases and refactored!